src/multiple-choice/components/runtime.tsx

```diff
@@ -69,7 +69,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
           authoredState.choices && authoredState.choices.map(choice => {
             const checked = selectedChoiceIds.indexOf(choice.id) !== -1;
             return (
-              <div key={choice.id} className={report && getChoiceClass(choice, questionScored, checked)}>
+              <div key={choice.id} className={report ? getChoiceClass(choice, questionScored, checked) : undefined}>
```

Comment:
Not sure it is worth it, but you could move `getChoiceClass` into the Runtime function so getChoiceClass would have direct access to the questionScored and report variables.  Then this logic could be moved out of jsx.